cmd/argo/commands/common.go

```diff
@@ -71,6 +72,9 @@ func initializeSession() {
 		wfv1.NodeFailed:    ansiFormat("✖", FgRed),
 		wfv1.NodeError:     ansiFormat("⚠", FgRed),
 	}
+	nodeTypeIconMap = map[wfv1.NodeType]string{
+		wfv1.NodeTypeSuspend: ansiFormat("ǁ", FgCyan),
```

Comment:
Hmmm I agree that `ǁ` is probably not idea. How about U+23F8: ⏸. Here it shows up as an emoji but I'm curious if in the terminal it will not. If it still shows up as an emoji we can find something else.

Reply:
Curious how this will show up in a code block: `⏸`

Reply:
I tested with my terminal and it simply displayed as is, but I am not sure if it can be displayed properly on all platform. Another problem with this block is it's full-width, which means we might need an additional space character but this would break the alignment.

And for me the orange color is disturbing.
![image](https://user-images.githubusercontent.com/4591982/74589697-602f5600-5042-11ea-9c8b-bb1e53bcc846.png)

Reply:
Hmm that's not ideal then. What about these options, in order of my preference?

- U+29B7: `⦷`? Looks like this is probably ideal: 

![image](https://user-images.githubusercontent.com/614838/74591465-2112ed80-4fcd-11ea-854f-2fe095e7200f.png)


- U+25CD: `◍`? Supposed to be a white circle with vertical bars, but might be hard to see at this size: 

![image](https://user-images.githubusercontent.com/614838/74591430-c083b080-4fcc-11ea-83b6-d225f908df37.png)

- U+25CC: `◌`, but I think this might be a better icon for skipped nodes than suspended nodes.

![image](https://user-images.githubusercontent.com/614838/74591472-3720ae00-4fcd-11ea-9602-94ea0ccdb9df.png)

Reply:
I think U+29B7 is worse. Since the bars are surrounded by a circle, they get smaller and thus visually closer to each other. They are thinner so the look darker. Here's a sample, roughly the same as what we see in your comment:
![image](https://user-images.githubusercontent.com/4591982/74591941-81e70800-5057-11ea-9feb-5dd86fdedda1.png)


Although the display depends on fonts, in general I think `ǁ` is still clearer than `⦷`. This screenshot was captured with font "monospace" at size 12, which is closer to what we expect (note the slight difference in alignment compared with the screenshot in PR description):
![image](https://user-images.githubusercontent.com/4591982/74591862-92e34980-5056-11ea-9fa9-3f7dc4d62098.png)
(The screenshot in the PR description was captured with font "Noto mono regular" at size 12.)